v1.0/Specification.md

```diff
@@ -138,7 +138,7 @@ The following table describes the requirements for data quality, when preserving
 | --- | --- | --- |
 | CSV_1 | CSV file format | The file **MUST** comply with the specification “Common Format and MIME Type for Comma-Separated Values (CSV) Files, RFC 4180”. |
 | CSV_2 | Extension | The file **MUST** have extension “.csv”. |
-| CSV_3 | Sheets | The spreadsheet **MUST** have each sheet converted to an independent file named “Sheet1”, “Sheet2” etc. |
+| CSV_3 | Sheets | The spreadsheet **MUST** have each sheet converted to an independent file named after the sheet's name |
```

Comment:
Every worksheet in a spreadsheet MUST be converted and exported to a single file named consistently with the sheet's name and a file extension consistent with the converted format.

Reply:
Even the above is dubious in that it MAY be possible to have sheets with the same name in some circumstances, what then?